README.md

```diff
@@ -180,6 +180,34 @@ room {
 * There can only be a single schema export directory for the project - you cannot configure variant-specific export
 directories.  Schemas exported from different variants will be merged in the directory specified in the "room" extension.
 
+#### Ksp
+Since version 2.6.6, `RoomSchemaLocationWorkaround` supports Kotlin Symbol Processing(Ksp). Like kapt, applying the Room
+processor with Ksp was causing cache misses. The workaround allows you to specify an output directory for Room schema
+exports without breaking caching. The schema export directory must be configured via the "room" project extension instead
```

Comment:
I think the word "breaking caching" is too vague. Let's be more specific here.

Reply:
removed the statement and updated the version to 2.7.0